src/08_comprehensions.py

```diff
@@ -11,31 +11,36 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
-print (y)
+for x in range(6):
```

Comment:
these aren't list comprehensions `x = [x for x in range(3)]` this should result in this [0, 1, 2]. This is the idea of list comprehension: making code more readable.